apps/landing/features/aboutUs/MemberCard.tsx

```diff
@@ -0,0 +1,128 @@
+import { type FC, useState } from 'react';
+import { StyleSheet } from 'react-native';
+import { Hoverable, Text, View } from '@walless/gui';
+import Image from 'next/image';
+
+import type { SocialProps } from './Social';
+import SocialCard from './Social';
+
+export interface MemberProps {
+	name: string;
+	positions: string[];
+	avatar: string;
+	description: string;
+	socials: SocialProps[];
+	size?: number;
+}
+
+const MemberCard: FC<MemberProps> = ({
+	name,
+	positions,
+	avatar,
+	description,
+	socials,
+	size = 212,
+}) => {
+	const [isHovered, setIsHovered] = useState(false);
+	const BIGGEST_NUMBER_OF_CARD = 5;
+	const containerStyle = {
+		width: size,
+		height: (size * 312) / 362,
+	};
+
+	const containerMarginStyle = {
+		margin:
+			(1200 - size * BIGGEST_NUMBER_OF_CARD) / (BIGGEST_NUMBER_OF_CARD * 3),
+		borderRadius: size > 300 ? 20 : 12,
+	};
+
+	return (
+		<Hoverable
+			onHoverIn={() => setIsHovered(true)}
+			onHoverOut={() => setIsHovered(false)}
+			style={[styles.container, containerStyle, containerMarginStyle]}
+		>
+			<Image
+				style={{
+					objectFit: 'cover',
+				}}
+				src={avatar}
+				alt=""
+				width={size}
+				height={(size * 312) / 362}
+			/>
+			{isHovered && (
+				<View style={[styles.hoveredContainer, containerStyle]}>
+					<View style={styles.imageBackground}>
+						<Image
+							src="/img/team/walless-logo-mask.svg"
+							alt=""
+							width={(size  )}
+							height={(size  ) * 163 /314}
+						/>
+					</View>
+					<View>
+						<Text style={styles.nameText}>{name}</Text>
+						<Text style={styles.smallText}>
+							{positions.map((item) => {
+								if (item === positions[0]) {
+									return item;
+								} else {
+									return ` & ${item}`;
+								}
+							})}
+						</Text>
+					</View>
```

Comment:
avoid `View` tag without style if possible, try if it can be replaced by `Fragment`